proto/Screen.proto

```diff
@@ -40,10 +69,33 @@ message ContentNode {
   oneof content {
     NumericValueWidget numericValueWidget = 2;
     ColorWidget colorWidget = 3;
+    TemperatureWidget temperatureWidget = 4;
+    SetpointSensorPairWidget setpointSensorPairWidget = 5;
+    DigitalClockWidget digitalClockWidget = 6;
+    TitleWidget titleWidget =7;
   }
 }
 
-message Config {
-  repeated LayoutNode layoutNodes = 1 ;
+
+enum TemperatureUnit {
+  TEMP_CELSIUS = 0;
+  TEMP_FAHRENHEIT = 1;
+};
+
+
+message Page {
+  repeated LayoutNode layoutNodes = 1;
   repeated ContentNode contentNodes = 2;
+  string name = 3 [ (nanopb).max_size = 20 ];
+  uint32 id = 4;
+}
+
+
+message Block {
+  option (brewblox.msg).objtype = ScreenConfig;
+  repeated Page pages = 1;
+  string name = 2 [ (nanopb).max_size = 40 ];
+  TemperatureUnit tempUnit = 3;
```

Comment:
tempUnit / brightness / timeZone were moved to SysInfo, and do not have to be present here.